python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1194,6 +1194,13 @@ def forward_normal_core(self, q, k, v, forward_batch):
         output, _ = self.o_proj(attn_output)
         return output
 
+    def _enable_fuse_rope(self, forward_batch: ForwardBatch) -> bool:
```

Comment:
nit: maybe we can name it to fuse_rope_to_blahblah or sth like that, b/c today I also write some code about "fuse rope with set-kv-buffer" and it seems different from that